src/View/Helper/Factory/FormElementErrorsFactory.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Form\View\Helper\Factory;
+
+use Laminas\Form\View\Helper\FormElementErrors;
+use Psr\Container\ContainerInterface;
+
+use function array_key_exists;
+use function is_array;
+
+class FormElementErrorsFactory
```

Comment:
The other factories we have have signatures for `laminas-servicemanager`.
I see this can be a standalone class, but for sake of consistency what about implementing `\Laminas\ServiceManager\Factory\FactoryInterface` here too?

Reply:
Two reasons why I do not use it here:

* the features `$requestedName` and `$options` of the interface are not needed
* `Laminas\ServiceManager\Factory\FactoryInterface` still uses the interface of container-interop

Reply:
Ah, we can mark this class as final.

Reply:
Ok, I'll make most classes `final` soon in a separate PR